source/developers-guide/rest-api/api-resource-countries/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: REST API - Countries Resource
+github_link: developers-guide/rest-api/api-resource-countries/index.md
+indexed: false
```

Comment:
As discussed with @MarcelSchmaeing, it should be included in 5.2. Should I add `shopware_version: 5.2` instead?